shared/src/main/i18n/com/kuvaszuptime/kuvasz/i18n/Messages_pl.properties

```diff
@@ -0,0 +1,215 @@
+toggleNavigation=Przełącz nawigację  
+signOut=Wyloguj się  
+readOnlyNotice=Kuvasz działa w trybie tylko do odczytu, ponieważ monitory zostały skonfigurowane przez YAML. Możesz tylko przeglądać monitory, ale nie możesz ich edytować ani usuwać.  
+enableDarkMode=Tryb ciemny  
+enableLightMode=Tryb jasny  
+dashboard=Panel  
+monitors=Monitory  
+settings=Ustawienia  
+docs=Dokumentacja  
+login=Logowanie  
+sponsor=Sponsor  
+
```

Comment:
Please put back the comment lines to their original places to make the life of the future contributors easier 🙏